pkg/controller.v1beta1/trial/trial_controller.go

```diff
@@ -57,6 +58,8 @@ var (
 	log = logf.Log.WithName(ControllerName)
 	// errMetricsNotReported is the error when Trial job is succeeded but metrics are not reported yet
 	errMetricsNotReported = fmt.Errorf("metrics are not reported yet")
+	// errReportMetricsFailed is the error when metrics are reported but not successful yet
+	errReportMetricsFailed = fmt.Errorf("failed to report metrics")
```

Comment:
```suggestion
	// errReportMetricsFailed is the error when `unavailable` metrics value can't be inserted to the Katib DB.
	errReportMetricsFailed = fmt.Errorf("failed to report unavailable metrics")
```

Reply:
@Electronic-Waste Just a small nit from my side and we can merge it.

Reply:
@andreyvelich I've committed your suggestions :)